content/participate/index.html.haml

```diff
@@ -1,138 +1,196 @@
 ---
 layout: simplepage
-title: Participate and contribute
+title: Participate and Contribute
```

Comment:
I think this was intentionally lowercase.

Reply:
The site is quite consistently using title-case, so the new one is IMHO better.